README.md

```diff
@@ -1,17 +1,16 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# LEADERBOARD
```

Comment:
- [ ]  Since previous reviewer didn't raise this as required, I am marking this as optional, however, is strongly suggest you implement this, since in the next submission, you would be asked to do so.
- Kindly use [this updated README file](https://github.com/microverseinc/readme-template) provided by the Microverse. All main headers should from this template README file should be present in your repo. The only optional section is FAQ. From the section description, make sure you only have relevant sections present in your README file. Also, don't forget to remove the Microverse logo, optionally you can replace it with your own logo if you have one.